File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        
        int answer[] = new int[2];
        
        int gcbNum = gcb(n, m);
        int lcmNum = n*m/gcbNum;
        
        answer[0] = gcbNum;
        answer[1] = lcmNum;
        
        return answer;
    }
    
    public int gcb(int a, int b){
        if(a % b == 0) return b;
        else return gcb(b, a%b);
    }
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    
    var gcbNum = gcb(n, m);
    var lcmNum = n*m/gcbNum;
    
    var answer = [gcbNum, lcmNum];
    
    return answer;
}

function gcb(a , b){
    if(a%b == 0) return b;
    else return gcb(b, a%b);
}